README.md

```diff
@@ -748,13 +748,14 @@ pipeline.notifyOnFailure(PipelineNotification.chime({
 }));
 ```
 
-## ECR Registry Sync
+## ECR Mirror
```

Comment:
I've been struggling with this name originally as well. Its not really a mirror because it can build custom images. Also mirror gives the impressions its fully mirroring an upstream, which it isn't.

I landed on `EcrRegistry` since it basically represents a registry that users can push images to. The fact some of them are coming from `GitHub` is somewhat incidental now and is manifested in the `ImageSource` class, which i feel is the right place.

The fact it can periodically update the images is also just a registry feature, not necessarily related to any upstream.

WDTYT?

Reply:
I would avoid over-thinking the name and trying to over-generalize the set of features. What we're aiming to do here is to sync docker images into ECR.

How about removing the `fromDirectory()` API from delivlib since it's only needed for `cdk-ops`, and then the construct will more closely match an ECR Mirror construct.

Reply:
I think removing `fromDirectory` will be a little strange. The existence of the `MirrorSource` class and the `bind` API you've added makes it inherently extensible, which is great, but it is no longer just a mirror. 

> What we're aiming to do here is to sync docker images into ECR

This was the initial intent, but it grew because we needed it to, and it then ended up as a generic construct that allows pushing images to an ECR registry. 

In any case, I don't mind keeping the name, just wanted to offer my point of view.